step5.html

```diff
@@ -30,34 +30,14 @@
 rake -T
 {% endhighlight %}
 <p>
-It failed!&hellip; and with a rather cryptic error message. But don't worry, it's
-easy to fix.
-</p>
-<p>
-Many of the tasks provided by yast-rake are related to packaging (to be precise,
-those packaging tasks are actually provided by the
-<a href="https://rubygems.org/gems/packaging_rake_tasks">packaging_rake_tasks</a>
-gem, required by yast-rake). For that reason you need a package specification to
-get the YaST rake tasks running. In your case, that is as easy as executing the
-following command.
+Now, let's try some of the useful available tasks. But first we need to install
+"rubygem-gettex".
 </p>
 {% highlight bash %}
-git checkout package_spec
+zypper install ruby2.1-rubygem-gettext
```

Comment:
This specific command is in fact specific of your ruby version. In tumbleweed (or future versions of Leap) it would be
```
zypper install ruby2.2-rubygem-gettext
```